bloom-housing/bloom
# refactor: remove backend deps from what to expect

# Pull Request Template

## Issue Overview

This PR addresses #2776


## Description

Removes backend dependencies and pre-translated strings from the `WhatToExpect` component. I pulled over the improvements from Detroit where we added an optional expandable section.


Also renamed the `WhatToExpect` component to the `ExpandableSection` component, because it is just string and an expandable section - it doesn't need to be tied to `WhatToExpect`.

## How Can This Be Tested/Reviewed?

Check out the new stories, and ensure the what to expect section renders with regression.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
